cmake/GinkgoConfig.cmake.in

```diff
@@ -92,21 +92,17 @@ set(GINKGO_HAVE_HWLOC @GINKGO_HAVE_HWLOC@)
 set(GINKGO_HAVE_ROCTX @GINKGO_HAVE_ROCTX@)
 
 # Ginkgo installation configuration
-set(GINKGO_CONFIG_FILE_PATH "${CMAKE_CURRENT_LIST_DIR}")
-string(REPLACE "@GINKGO_INSTALL_CONFIG_DIR@" "" GINKGO_INSTALL_PREFIX "${GINKGO_CONFIG_FILE_PATH}")
-set(GINKGO_INSTALL_INCLUDE_DIR "${GINKGO_INSTALL_PREFIX}/@GINKGO_INSTALL_INCLUDE_DIR@")
-set(GINKGO_INSTALL_LIBRARY_DIR "${GINKGO_INSTALL_PREFIX}/@GINKGO_INSTALL_LIBRARY_DIR@")
-set(GINKGO_INSTALL_PKGCONFIG_DIR "${GINKGO_INSTALL_PREFIX}/@GINKGO_INSTALL_PKGCONFIG_DIR@")
-set(GINKGO_INSTALL_CONFIG_DIR "${GINKGO_INSTALL_PREFIX}/@GINKGO_INSTALL_CONFIG_DIR@")
-set(GINKGO_INSTALL_MODULE_DIR "${GINKGO_INSTALL_PREFIX}/@GINKGO_INSTALL_MODULE_DIR@")
+set(GINKGO_INSTALL_PREFIX "@PACKAGE_CMAKE_INSTALL_PREFIX@")
+set(GINKGO_INSTALL_INCLUDE_DIR "@PACKAGE_CMAKE_INSTALL_FULL_INCLUDEDIR@")
+set(GINKGO_INSTALL_LIBRARY_DIR "@PACKAGE_CMAKE_INSTALL_FULL_LIBDIR@")
```

Comment:
when build the ginkgo without installation, the GINKGO_INSTALL_INCLUDE_DIR and GINKGO_INSTALL_LIBRARY_DIR  will be wrong.

Reply:
in that case, there is no valid include dir, so I think we can ignore this edge case for now - the provided target should still work. Related: [CMP0090](https://cmake.org/cmake/help/latest/policy/CMP0090.html#policy:CMP0090)

Reply:
I usually use this way by giving the cmake location to the integration. It can ensure I use the ginkgo library is the latest one without `make install` when developing.

Reply:
It still works, as long as you use the installed target - most of the GinkgoConfig.cmake variables don't really provide correct paths for the exported setting anyways, so I think we can ignore that.